Objects/object.c

```diff
@@ -2917,12 +2910,7 @@ _Py_Dealloc(PyObject *op)
     // Make sure that type->tp_name remains valid
     Py_INCREF(type);
 #endif
-
-    struct _reftracer_runtime_state *tracer = &_PyRuntime.ref_tracer;
-    if (tracer->tracer_func != NULL) {
-        void* data = tracer->tracer_data;
-        tracer->tracer_func(op, PyRefTracer_DESTROY, data);
-    }
+    _PyReftracerTrack(op, PyRefTracer_DESTROY);
```

Comment:
Can you move this down a few lines next to the call to `(*dealloc)(op)`.
That way we can make sure it is eliminated when we fix https://github.com/python/cpython/issues/125790

Reply:
Sure, will do

Reply:
Done